src/secureSignalUid2.ts

```diff
@@ -1,3 +1,4 @@
+//@ts-nocheck
```

Comment:
I think the fact that we need to ignore typescript errors here is a sign our types aren't right - this is the same pattern we recommend for partners. Nothing to fix here, it's just a reminder that it might be worth looking into a fix at some point.